plugins/inventory/proxmox.py

```diff
@@ -136,6 +139,14 @@
   my_inv_var_1: "'my_var1_value'"
   my_inv_var_2: >
     "my_var_2_value"
+
+# Specify the url, user and password using templating
+# my.proxmox.yml
+plugin: community.general.proxmox
+url: "{{ lookup('ini', 'url', section='proxmox', file='file.ini') }}"
+user: "{{ lookup('env','PM_USER') | default('ansible@pve') }}"
```

Comment:
Ah, one tiny thing, we should use FQCN for builtin plugins as well:
```suggestion
url: "{{ lookup('ansible.builtin.ini', 'url', section='proxmox', file='file.ini') }}"
user: "{{ lookup('ansible.builtin.env','PM_USER') | default('ansible@pve') }}"
```